lib/bulk_data/cache.rb

```diff
@@ -22,6 +22,10 @@ def self.read(key)
       cache.read(key) || (raise NoEntryError)
     end
 
+    def self.populated?(key)
+      cache.exist?(key)
+    end
```

Comment:
Rather than explicitly adding a method I think it'd be preferable to delegate `exist?` to the cache instance: https://github.com/alphagov/content-publisher/blob/c14461000856ef638303eb63b1e4315d638e13ec/lib/bulk_data/cache.rb#L11

I think we also can kill of the clear method as that is already delegated.

Reply:
clear method is now gone :)